website/source/docs/guides/creating-certificates.html.md

```diff
@@ -25,159 +58,201 @@ signed by this CA will be allowed to communicate with the cluster.
 ~> Consul certificates may be signed by intermediate CAs as long as the root CA
    is the same. Append all intermediate CAs to the `cert_file`.
 
+### Step 1: Create a Certificate Authority
 
-## Reference Material
+There are a variety of tools for managing your own CA, [like the PKI secret
+backend in Vault][vault-pki], but for the sake of simplicity this guide will
+use Consul's builtin TLS helpers:
 
-- [Encryption](/docs/agent/encryption.html)
+```shell
+$ consul tls ca create
+==> Saved consul-ca.pem
+==> Saved consul-ca-key.pem
+```
 
-## Estimated Time to Complete
+The CA certificate (`consul-ca.pem`) contains the public key necessary to
+validate Consul certificates and therefore must be distributed to every node
+that requires access.
 
-20 minutes
+~> The CA key (`consul-ca-key.pem`) will be used to sign certificates for Consul
+nodes and must be kept private.
 
-## Prerequisites
+### Step 2: Create individual Server Certificates
 
-This guide assumes you have [cfssl][cfssl] installed (be sure to install
-cfssljson as well).
+Create a server certificate for datacenter `dc1` and domain `consul`, if your
+datacenter or domain is different please use the appropriate flags:
 
-## Steps
+```shell
+$ consul tls cert create -server
+==> WARNING: Server Certificates grants authority to become a
+    server and access all state in the cluster including root keys
+    and all ACL tokens. Do not distribute them to production hosts
+    that are not server nodes. Store them as securely as CA keys.
+==> Using consul-ca.pem and consul-ca-key.pem
+==> Saved consul-server-dc1-0.pem
+==> Saved consul-server-dc1-0-key.pem
+```
 
-### Step 1: Create Certificate Authority
+Please repeat this process until there is an *individual* certificate for each
+server. The command can be called over and over again, it will automatically add
```

Comment:
I had similar thoughts. Since it is mentioned in our security model I thought I add that to our guide as well.

> . At a minimum, verify_outgoing should be enabled to verify server authenticity with each server having a unique TLS certificate.

Plus as you said the user might be able to revoke at some point. I don't feel strong about it, but leaning towards leaving it in.